### DISCUSSION TITLE
Driver/ID list for command line arguments

### DISCUSSION THREAD

**Discussion by JMajeau** [dbeaver/dbeaver#17416] [Q&A] [answered]
Hello,

I'm not sure this is a feature request, it's more of a question.
In your documentation (https://dbeaver.com/docs/wiki/Command-Line/), there is a section called **Connection Parameters**. That section shows how to create a single command line argument to connect to a database. In it’s parameters, there is the non-optional **driver (Driver name or ID)**. I was wondering if it was possible to get the list of keywords you use for that section of the argument.

**Comment by ShadelessFox** [comment, marked as answer]
Hello @JMajeau,

If I understand you correctly, you can find the ID of the desired driver by looking it up in the Driver Manager:

![image](https://user-images.githubusercontent.com/35821147/185414063-1c9eb20d-e694-4ebe-9e94-637e4d884460.png)

**Comment by JMajeau** [comment]
This does seem to be what I was looking for, thank you!
